programs/lfx-mentorship/2024/01-Mar-May/project_ideas.md

```diff
@@ -18,4 +18,33 @@
 
 ## Proposed Project ideas
 
+### Jaeger
+
+#### Jaeger-V2 Storage Backends
+
+- Description: Jaeger is a distributed tracing platform. Jaeger V2 is a major new version where we rebase all Jaeger backend components (agent, collector, ingester, and query) on top of the OpenTelemetry Collector. Currently only memory storage is wired in v2, we need to add Elasticsearch, Opensearch, Cassandra, Badger.
+- Expected Outcome: Build out full support in jaeger-v2 for all storage backends supported by jaeger-v1
+- Recommended Skills: Go, scripting, CI/CD
+- Mentor(s):
+  - Yuri Shkuro (@yurishkuro, github@ysh.us)
```

Comment:
Could you please add a back up mentor? This year, we really encourage having 2 mentors in all programs.

Reply:
added